src/main/java/at/hannibal2/skyhanni/config/features/dev/HideFarEntitiesConfig.java

```diff
@@ -0,0 +1,18 @@
+package at.hannibal2.skyhanni.config.features.dev;
+
+import com.google.gson.annotations.Expose;
+import io.github.moulberry.moulconfig.annotations.ConfigEditorBoolean;
+import io.github.moulberry.moulconfig.annotations.ConfigEditorSlider;
+import io.github.moulberry.moulconfig.annotations.ConfigOption;
+
+public class HideFarEntitiesConfig {
+    @Expose
+    @ConfigOption(name = "Enabled", desc = "Hide all entities from rendering except the nearest ones.")
+    @ConfigEditorBoolean
+    public boolean enabled = false;
+
+    @Expose
+    @ConfigOption(name = "Amount", desc = "Keep showing this amount of nearest entities.")
+    @ConfigEditorSlider(minValue = 5f, maxValue = 150f, minStep = 5f)
+    public float amount = 50f;
```

Comment:
Why not just have this be an int straight away?

Reply:
i'm guessing because sliders sometimes can act weird with non floats